modules/machineset-yaml-aws.adoc

```diff
@@ -145,23 +171,34 @@ $ oc -n openshift-machine-api \
 Custom tags can also be specified during installation in the `install-config.yml` file. If the `install-config.yml` file and the machine set include a tag with the same `name` data, the value for the tag from the machine set takes priority over the value for the tag in the `install-config.yml` file.
 ====
 
+ifndef::edge[]
 <6> Specify the zone, for example, `us-east-1a`.
```

Comment:
We seem to almost have duplicate entries for specifying a zone. Can we merge into one callout entry?

> Specify each zone, for example, `us-east-1a`, `us-east-1-nyc-1a`, and so on.